src/main/scala/beam/utils/matsim_conversion/MatsimPlanConversion.scala

```diff
@@ -147,7 +147,7 @@ object MatsimPlanConversion {
         n match {
           case elem: Elem if elem.label == "person" =>
             val attrs = elem.attributes
-            val filteredAttrs = attrs.filter(_.key.equals("id"))
+            val filteredAttrs = attrs.filter(m => m.key.equals("id"))
```

Comment:
As this is not using a meaningful name I see this as just adding white noise. Unless we are stepping away from using underscores altogether?